Complete de milliseconds feature (#3672)
telefonicaid/fiware-orion: doc/manuals/admin/database_model.md

    -   **mdNames**: an array of strings. Its elements are the names of the
        metadata of the attribute. Here the "." to "="
        replacement is not done.

>     -   **creDate**: the timestamp (as float number, meaning seconds with milliseconds)

Reviewer: ```suggestion
    -   **creDate**: the timestamp (as a floating point number, meaning seconds with milliseconds)
```

        corresponding to attribute creation (as a consequence of append).

>     -   **modDate**: the timestamp (as float number, meaning seconds with milliseconds)

Reviewer: ```suggestion
    -   **modDate**: the timestamp (as a floating point number, meaning seconds with milliseconds)
```

        corresponding to last attribute update. It matches creDate if the attribute has
        not been modified after creation.
-   **attrNames**: an array of strings. Its elements are the names of the
    attributes of the entity (without IDs). In this case, the "." to "="
    replacement is not done.

> -   **creDate**: the timestamp (as float number, meaning seconds with milliseconds)

Reviewer: ```suggestion
-   **creDate**: the timestamp (as a floating point number, meaning seconds with milliseconds)
```

    corresponding to entity creation date (as a consequence of append).

> -   **modDate**: the timestamp (as float number, meaning seconds with milliseconds)

Reviewer: ```suggestion
-   **modDate**: the timestamp (as a floating point number, meaning seconds with milliseconds)
```

>     corresponding to last entity update. Note that it uses to be the same that a

Reviewer: ```suggestion
    corresponding to last entity update. Note that it is usually the same as a
```
